pgroll-integration.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Pgroll Integration
+description: 'The pgroll integration behind our latest public beta'
+author: Emily Morgan
+authorEmail: emily@xata.io
+date: 03-14-2024
+tags: ['engineering', 'pgroll']
+published: true
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pgroll-integration/any.jpg
```

Comment:
What should the header image be?

Reply:
Elizabet just gifted everyone with their blog post images haha. We can use the one here: https://github.com/xataio/mdx-blog/issues/211#issuecomment-1997658649